libpod/events.go

```diff
@@ -136,6 +136,19 @@ func (c *Container) newExecDiedEvent(sessionID string, exitCode int) {
 	}
 }
 
+// newNetworkEvent creates a new event based on a network create/remove
+func (r *Runtime) NewNetworkEvent(status events.Status, netName, netID, netDriver string) {
+	e := events.NewEvent(status)
+	e.Network = netName
+	e.ID = netID
+	e.Attributes = make(map[string]string)
+	e.Attributes["Driver"] = netDriver
```

Comment:
this needs to be `"driver"` as you match `e.Attributes["driver"]` below and I think using lower case looks better here

Reply:
Thanks for the pointer,, somehow this was missed. I have updated the code and squashed all commits.